thefuck/rules/python_module_error.py

```diff
@@ -0,0 +1,13 @@
+import re
+from thefuck.shells import shell
+
+MISSING_MODULE = r"No module named '([a-zA-Z_][a-zA-Z0-9_]*)'"
+
+
+def match(command):
+    return "ModuleNotFoundError" in command.output
+
+
+def get_new_command(command):
+    missing_module = re.findall(MISSING_MODULE, command.output)[0]
+    return shell.and_("pip install {}".format(missing_module), command.script)
```

Comment:
```suggestion
    return shell.and_(u"pip install {}".format(missing_module), command.script)
```

Nit: add a unicode test case for `get_new_command` with `# -*- coding: utf-8 -*-` file header.

Reply:
Umm, I'll need help understanding this one. I'll integrate other changes first, then I'll come back to this.

Reply:
I implemented and pushed the other suggestions. Could you please explain this suggestion a bit more? I didn't understand it completely.

Reply:
That's actually not necessary, my bad.

It would probably be nice to have a test case for a module with special chars — i.e. `from café import cafeína` — which would then require the strings to be defined and treated as unicode, in order to work on Python 2.7. But then, alas, Python 2.7 doesn't support that. Moreover, the error is another one entirely — `SyntaxError: invalid syntax`.

Reply:
So should I resolve the conversation?